istio/resources/agent-config-map.yaml

```diff
@@ -0,0 +1,86 @@
+kind: ConfigMap
+metadata:
+  name: grafana-agent
+  namespace: istio-system
+apiVersion: v1
+data:
+  agent.yaml: |
+    metrics:
+      wal_directory: /var/lib/agent/wal
+      global:
+        scrape_interval: 60s
+        external_labels:
+          cluster: minikube
+      configs:
+      - name: integrations
+        remote_write:
+        - url: https://prometheus-us-central1.grafana.net/api/prom/push
+          basic_auth:
+            username: 671265
+            password: eyJrIjoiMjdlNjUxNmFjMzJhMTRhNmY1YTkyYjc1YjFiYTEyZmE5NDMwZjUyNiIsIm4iOiJzdGFjay00ODUyNDUtZWFzeXN0YXJ0LXByb20tcHVibGlzaGVyIiwiaWQiOjc1MjQ5MX0=
```

Comment:
Might want to revert this, and/or rotate your creds.

Reply:
@rgeyer removed this file